docs/registers/pp.md

```diff
@@ -9,15 +9,19 @@ comments: true
 # Parallel Input Parallel Output
 {: .no_toc }
 
+## Table of contents
+{: .no_toc .text-delta }
 
+1. TOC
+{:toc}
 
 
 ## Introduction
  
-In this mode, the 4 bit binary input B0, B1, B2, B3 is applied to the data inputs D0, D1, D2, D3 respectively of the four flip-flops. 
+In this mode, the 4-bit binary input B0, B1, B2, B3 is applied to the data inputs D0, D1, D2, D3 respectively of the four flip-flops. 
```

Comment:
content still from this source after refactor
https://www.tutorialspoint.com/computer_logical_organization/pdf/digital_registers.pdf

Reply:
Can I refer the sources which are listed in the references, and take content from there?